test/helpers/installer.py

```diff
@@ -40,6 +40,10 @@ class InstallerSteps(UserDict):
     _steps_jump[REVIEW] = PROGRESS
     _steps_jump[PROGRESS] = []
 
+    _parent_steps = {}
+    _parent_steps[DISK_ENCRYPTION] = DISK_CONFIGURATION
```

Comment:
We can get parent steps automatically. In fact we already do it.

previous_page = [k for k, v in self.steps._steps_jump.items() if current_page in v][0]

Reply:
This gets previous page (INSTALLATION_METHOD), not parent page (DISK_CONFIGURATION). Parent page is the page/step with substeps which needs to be clicked on the sidebar to open the substeps and make them clickable.